test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +36,31 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception {
+        // contains null
+        assertContainsNull(null, null);
+        assertContainsNull(null, "a", "b", null);
+
+        // does not contains null
+        assertNotContainsNull("abc", "", "abc", "ABC");
+        assertNotContainsNull("", "abc", "a", "abc");
+        assertNotContainsNull(1, Integer.valueOf(1));
+    }
+
     private void assertAreUnique(Object... objects) {
         assertTrue(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
 
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
-}
+
+    private void assertContainsNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
```

Comment:
No need to abstract the `assertTrue(Utils.isAnyNull(...))` as separate method, because asserting the result of the method itself is fine.

Although existing test `assertAreUnique` does that, but that is to prevent `Arrays.asList(...)` from flying everywhere.